apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V75__new_schema_remove_old_claims_tables.sql

```diff
@@ -0,0 +1,27 @@
+--
+-- Script to drop old claims tables that have been replaced by _new tables.
+-- Once all db changes are complete, the _new table names will be renamed
+-- back to their original table names.
+--
+DROP TABLE IF EXISTS public.partd_events CASCADE;
+--
+DROP TABLE IF EXISTS public.carrier_claim_lines CASCADE;
+DROP TABLE IF EXISTS public.carrier_claims CASCADE;
```

Comment:
Wont cascading the main claims table with cascade also drop its lines table due to the relationship?